cms/templates/settings.html

```diff
@@ -1,3 +1,4 @@
+<%page expression_filter="h"/>
 <%inherit file="base.html" />
 <%def name="online_help_token()"><% return "schedule" %></%def>
 <%block name="title">${_("Schedule &amp; Details Settings")}</%block>
```

Comment:
Can you replace `&amp;` with `&` ?

Also I am working on making this template settings.html safe in the following [PR#12042](https://github.com/edx/edx-platform/pull/12042) and the files it is using. So, I think it would be worth that you leave this work to me. 

I also see that you haven't added any new HTML/javascript to the code, so it would be same in my PR.

@cahrens what do you think ?